cluster/manifests/skipper/deployment.yaml

```diff
@@ -541,6 +542,7 @@ spec:
           - "-reverse-source-predicate"
           - "-default-filters-dir=/etc/config/default-filters"
           - '-default-filters-prepend={{ .Cluster.ConfigItems.skipper_default_filters }}'
+          - '-default-filters-append={{ .Cluster.ConfigItems.skipper_default_filters_authentication }}'
```

Comment:
why creating a new config item for something we already have?
`skipper_default_filters_append` is already there